packages/blade/src/components/Input/BaseInput/AnimatedBaseInputWrapper.native.tsx

```diff
@@ -137,7 +146,7 @@ const _AnimatedBaseInputWrapper: React.ForwardRefRenderFunction<
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       ref={ref as any}
       style={[
-        isDropdownTrigger
+        isDropdownTrigger && !isTextArea
```

Comment:
🤔 ummm why? add a comment?

Reply:
> 🤔 ummm why?

Good question. I also forgot. Lemme remember and add comment